binary/grakn

```diff
@@ -17,7 +17,9 @@
 #
 
 # Grakn global variables
-JAVA_BIN=java
+JAVA_HOME=${JAVA_HOME:-java}
+echo "Using java executable at $JAVA_HOME"
+JAVA_BIN=$JAVA_HOME
```

Comment:
@haskie-lambda usually `JAVA_HOME` is pointed at the root of JDK directory; therefore, `$JAVA_BIN` (Java's binary) would be `$JAVA_HOME/bin/java`

Reply:
right;
this should do the trick;
I can't set `$JAVA_BIN` because it might become `java` (from the original implementation)

I added a check to add "/bin/java" to the end of the path if it was configured